contract/src/test/java/io/neow3j/contract/TransactionBuilderTest.java

```diff
@@ -379,120 +489,115 @@ public void signTransactionWithAdditionalSigners() throws IOException {
     }
 
     @Test
-    public void failBuildingInvocationBecauseWalletDoesntContainSignerAccount()
-            throws IOException {
+    public void failBuildingTransactionBecauseWalletDoesntContainSignerAccount()
+            throws Throwable {
 
         Wallet w = Wallet.createWallet();
         Account signer = Account.createAccount();
         setUpWireMockForCall("invokescript", "invokescript_name_neo.json", SCRIPT);
-        Invocation.Builder b = new Invocation.Builder(neow)
-                .withScript(Numeric.hexStringToByteArray(SCRIPT))
-                .withWallet(w)
-                .withSigners(Signer.calledByEntry(signer.getScriptHash()))
-                .withValidUntilBlock(1000); // Setting explicitly so that no RPC call is necessary.
-        exceptionRule.expect(InvocationConfigurationException.class);
+        TransactionBuilder b = new TransactionBuilder(neow)
+                .script(Numeric.hexStringToByteArray(SCRIPT))
+                .wallet(w)
+                .signers(Signer.calledByEntry(signer.getScriptHash()))
+                .validUntilBlock(1000); // Setting explicitly so that no RPC call is necessary.
+        exceptionRule.expect(TransactionConfigurationException.class);
         exceptionRule.expectMessage(
-                new StringContains("Wallet does not contain the account for signer" +
-                        " with script hash " + signer.getScriptHash()));
-        b.build();
+                new StringContains("Can't create transaction signature. Wallet does not " +
+                        "contain the signer account with script hash " + signer.getScriptHash()));
+        b.buildTransaction();
     }
 
     @Test
-    public void failSigningInvocationBecauseWalletDoesntContainSignerAccount()
-            throws IOException {
+    public void failSigningTransactionBecauseWalletDoesntContainSignerAccount()
```

Comment:
This tests the same as `failBuildingTransactionBecauseWalletDoesntContainSignerAccount()`. They are both expected to fail in the `getSignerAccounts()` method.
Remove one of them.

Reply:
I'll keep the test with `.buildTransaction()` and remove the other.